### ISSUE TITLE
Login password is not suppressed in logs

### ISSUE THREAD

**Issue by dhoer** [dhoer/chef-macosx_gui_login#1]
The execute resource has sensitive attribute set to true, but it is still outputs the command:

```
     * execute[switch to login window] action run
         - execute sudo "/System/Library/CoreServices/Menu Extras/User.menu/Contents/Resources/CGSession" -suspend

         - execute           osascript -e '
            tell application "System Events"
              keystroke "vagrant"
              keystroke return
              delay 3.0
              keystroke "vagrant"
              delay 3.0
              keystroke tab
              keystroke return
              keystroke return
            end tell'
```

**Comment by dhoer** [comment]
https://github.com/chef/chef/issues/3749

**Comment by dhoer** [comment]
Sensitive is only available on Chef 12: chef/chef#3749
